### PR TITLE
#Fibo-24 initial django project

### PR DESCRIPTION
**Pull request by XSnelliusX**

### CODE REVIEW

File: backend/api/models.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    item_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=55)
+    amount = models.FloatField()
+    cashflow = models.ForeignKey(Cashflow, on_delete=models.CASCADE, blank=True, null=True)
+    value = models.FloatField()
+
+
+class Account(models.Model):
+    account_id = models.CharField(max_length=55, primary_key=True)
+    cashflow = models.ForeignKey(Cashflow, on_delete=models.CASCADE)
```
Comment (jens1o) [inline backend/api/models.py:36]: As indicated in the ER diagram, an account may have many cashflows, but one cashflow may only have one account. Hence, the foreign key should lie in the `Cashflow`, and not in the `Account`

File: backend/api/models.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    category_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=55)
+
+
+class Cashflow(models.Model):
+    cashflow_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    is_income = models.BooleanField()
+    overall_value = models.FloatField()
+    timestamp = models.DateTimeField(default=timezone.now)
+    categories = models.ManyToManyField(Category)
```
Comment (jens1o) [inline backend/api/models.py:22]: I hope I have resolved your question, here?

**Review by jens1o — CHANGES_REQUESTED** [review]
Thank you! I added one question, but looks great overall!

File: backend/api/models.py (hunk 2 above)
Comment (XSnelliusX) [reply backend/api/models.py:22]: Im not sure if this is what we want, becaus in our ER-Diagramm the Relation between Cashflow and Category is one to many and not many to many. The question was related to the "models.DO_NOTHING" 😅

**Comment by jens1o** [comment]
Is this ready for meging?

**Review by jens1o — APPROVED** [review]
great for now, needs to be updated in the future though. As the ER model update is a different ticket, let's merge this first and modify it in another pull request

**Pull request merged**